course_content/cartopy_course/solutions/cartopy_exercise_3

```diff
@@ -0,0 +1,41 @@
+
+# Cartopy course, exercise 3
+
+# Use set_extent to control plot : what coordinate system does this use ?
+
+# Repeat the 'correct' plot with a latlon-straight line, but control the map region.
+proj = ccrs.EquidistantConic()
+x0, y0 = -50, -30
+x1, y1 = 10, 55
+
+# Extend corners slightly
+ax = plt.axes(projection=proj)
+ax.coastlines()
+
+plt.plot([x0, x1], [y0, y1], linewidth=4, transform=ccrs.PlateCarree())
+
+# Use "set_extent", with slight margins added to the region corners.
+x0a = x0 - 0.1 * (x1 - x0)
+x1a = x1 + 0.1 * (x1 - x0)
+y0a = y0 - 0.1 * (y1 - y0)
+y1a = y1 + 0.1 * (y1 - y0)
+ax.set_extent([x0a, x1a, y0a, y1a])
+
+plt.show()
+
+#
+# **Also**
+# draw something roughly similar, but in map-coordinates instead of lats+lons.
```

Comment:
The problem with setting up the question this way is that the user is unlikely to have come up with this themself.

Reply:
Ok, I'm going to remove the second part for now.

There is space here for more exploration. It might be a good idea to show how set_extent delivers the region specified in other coords -- i.e. minimum map rectangle enclosing the target area.  But my attempt was getting too complicated.  We can re-visit it.